### ISSUE TITLE
Adding a property to Credentials with script

### ISSUE THREAD

**Issue by datacatsuk** [otykier/TabularEditor#604]
I have created some automation scripts to deploy models to multiple servers at once.
I am applying some piece of script before the deployment. One of the reason for that is to set credentials for SQL data source.
I am working with other developers who insist (for some inexplicable reason) on using Visual Studio for model development and deployment. When they edit the files, Password property of Credentials objects get deleted.
When I try to run that script below, I get the following error message.

> Object reference not set to an instance of an object.

![image](https://user-images.githubusercontent.com/20473345/92609506-8e120b80-f2ae-11ea-8ac4-a2d81fcdb1fc.png)

The script I'm using is below. It's the password line getting the error.

```
var source = (Model.DataSources["SQL/dwh database windows net;MyDB"] as StructuredDataSource);

source.Username = "xxxx";
source.Password = "yyyy";
source.Database = "MyDB";
```

Is there any way I can add this Password property if it's missing with a script piece?

**Comment by debashish10** [comment]
Suddenly today I have been getting the exact same error .

**Comment by otykier** [comment]
Thanks for reporting. I located a bug in the StructuredDataSource property setters in TOMWrapper and will provide a fix shortly. Just awaiting confirmation that #606 is the same issue.

**Comment by otykier** [comment]
Fixed in [2.12.3](https://github.com/otykier/TabularEditor/releases/latest)

**Comment by datacatsuk** [comment]
Thanks a lot for prompt action @otykier . Appreciate it!
